tests/e2e/test_cases_data/mnpt_data.hocon

```diff
@@ -0,0 +1,29 @@
+# test_data.hocon
+# Input/output test pairs
```

Comment:
Don't abbreviate leaving people guessing as to what this file is for.
No one will instantly know what mnpt actually means.